lib/Pages/ViewItem.dart

```diff
@@ -74,8 +75,7 @@ class _ViewItemState extends State<ViewItem> {
       date = item.date;
       owner = item.owner;
 
-      // TODO fetch from user database
-      email = user!.email;
+      email = provider.user?.email ?? "";
```

Comment:
Wait this should be the email of the owner of the item, not the current user's email